### DISCUSSION TITLE
Drawer OverlayBackground Corner Radius

### DISCUSSION THREAD

**Discussion by Jason37174** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3263] [Q&A]
How do I set the corner radius of the overlaybackground of a drawer? My main window has rounded corners and whenever I open the drawer the overlay is visible outside the main window. 

See sample image for details. I'm assuming it can be done with some sort of style I can apply to the drawerhost but I'm struggling to figure it out. The drawer itself, has a corner radius so that looks fine, but not the overlay layer.

Thanks for any help!

![Cornerradius on DrawerHost Overlay 1](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/assets/78761991/140498df-4cb8-40ef-a9ea-fc25334122ca)

**Comment by Keboo** [comment]
Can you try setting `ClipToBounds=True` on the DrawerHost and/or Window level and see if that makes a difference?

If possible, can you share your custom window chrome setup so we can test with it?
